app/javascript/packs/puerto-rico-hint.ts

```diff
@@ -0,0 +1,14 @@
+import { t } from '@18f/identity-i18n';
```

Comment:
since there is now a separate ticket for the pr address work do we want to pull this file out of this pr and save in a diff. branch?

Reply:
Puerto Rico hint text changes removed.